src/button/config.js

```diff
@@ -144,6 +147,19 @@ export const BUTTON_CONFIG : ButtonConfig = {
         allowUnbranded:  true
     },
 
+    [ BUTTON_LABEL.INSTALLMENT ]: {
+            label:     `{ content: installment }`,
+            logoLabel: `{ logo: ${ BUTTON_LOGO.PP } } { logo: ${ BUTTON_LOGO.PAYPAL } }`,
+
+            allowPrimary: true,
+            allowPrimaryVertical:   false,
+            allowPrimaryHorizontal: true,
+            allowSecondaryVertical:   false,
+            allowSecondaryHorizontal: false,
+            requiresTextWrap: true,
+            allowedDynamicContent: { 'installmentnum': 'installment_dynamic' }
```

Comment:
This abstraction is cool, but I'm a little worried that it might be a bit specific to the current use case; i.e. "use content A or content B depending on another style key", and that might not scale to other use cases.

Would it be possible instead to make `label` a function? Something like:

```
label: (props.style) => `{ content: ${ props.style.installmentperiod ? 'installment' : 'installment_period' } }`
```